openwisp_monitoring/device/api/views.py

```diff
@@ -200,6 +201,22 @@ def _write(self, request, pk):
                 metric.write(client['mac'])
             if created:
                 self._create_clients_graph(metric)
+        if 'resources' in data:
+            resources = {}
+            if 'load' in data['resources']:
+                resources['load'] = 100 * data['resources']['load'][1]
+            if 'disk_usage' in data['resources']:
+                used_space, total_size = 0, 0
+                for disk in data['resources']['disk_usage']:
+                    used_space += disk['used_space']
+                    total_size += disk['size']
+                resources['disk_usage'] = 100 * used_space / total_size
```

Comment:
I think that by doing this we are wasting the data sent by the monitoring script here.

Imagine you are an operator wanting to know the same information that you would get by ssh'ing into the device and just get an average. Some mount points are full and you don't even notice (!)

Please just save all the data sent from the monitoring script and don't early optimize that much. We can change it later. I propose storing load_1, load_5 and load_15 and just enable by now automatically the load_5 graph. WDYT?

Reply:
@PabloCastellano, regarding warning the `operator` I had a suggestion. How's this, if disk_usage of a `mounted_space` exceeds a `threshold` say `80%` we notify the user displaying the `mounted_space` name and `used` percentage. In my opinion it won't be a good idea to display all the `mounted points` on the `graph` because they might just be too many. It's 7 in my case and will vary as per device (also `charts` don't support dynamic values currently, if I am not wrong).

Reply:
I would say let's save the raw data but for the moment focus on displaying only the total like you're doing now.

In the future when we want to start displaying more data, we'll have the data already.

> also charts don't support dynamic values currently, if I am not wrong

Yes it's supported with the usage of `top_fields`, if you specify `top_fields` and set it to 10, it will do a query to get the 10 keys with most results and will then build the final query to use those fields.

Reply:
I would say let's save the raw data but for the moment focus on displaying only the total like you're doing now.

In the future when we want to start displaying more data, we'll have the data already.

> also charts don't support dynamic values currently, if I am not wrong

Yes it's supported with the usage of `top_fields`, if you specify `top_fields` and set it to 10, it will do a query to get the 10 keys with most results and will then build the final query to use those fields.